src/relay/volcanomaze.2.js

```diff
@@ -28,16 +28,24 @@ function adjustPlatforms(res) {
 				t.removeClass('no').addClass('you');
 				setTitle(t, 'You');
 			}
+		        t.removeClass('next');
+			if (t.attr('rel') == res.next) {
```

Comment:
curious looking indentation here (tabs + space) -- if this wasn't copied from KoL, might be nice to make equivalent?

Reply:
That was my editor. I will fix.

Reply:
Done.